discovery-provider/src/tasks/index_oracles.py

```diff
@@ -23,17 +23,21 @@
 
 
 def get_oracle_addresses_from_chain(redis) -> List[str]:
-    eth_rewards_manager_address = eth_registry_instance.functions.getContract(
-        bytes("EthRewardsManagerProxy", "utf-8")
-    ).call()
-    eth_rewards_manager_instance = eth_web3.eth.contract(
-        address=eth_rewards_manager_address, abi=REWARDS_CONTRACT_ABI
-    )
-    oracle_addresses = (
-        eth_rewards_manager_instance.functions.getAntiAbuseOracleAddresses().call()
-    )
-    redis.set(oracle_addresses_key, ",".join(oracle_addresses))
-    return oracle_addresses
+    try:
+        # Note: this call will fail until the eth rewards manager contract is deployed
+        eth_rewards_manager_address = eth_registry_instance.functions.getContract(
+            bytes("EthRewardsManagerProxy", "utf-8")
+        ).call()
+        eth_rewards_manager_instance = eth_web3.eth.contract(
+            address=eth_rewards_manager_address, abi=REWARDS_CONTRACT_ABI
+        )
+        oracle_addresses = (
+            eth_rewards_manager_instance.functions.getAntiAbuseOracleAddresses().call()
+        )
+        redis.set(oracle_addresses_key, ",".join(oracle_addresses))
+        return oracle_addresses
+    except Exception:
```

Comment:
it might be worth 1 liner log statement just in case for some reason this does fail for a web3 reason